HotSpot/View Controllers/EditInformationViewController.m

```diff
@@ -0,0 +1,116 @@
+//
+//  EditInformationViewController.m
+//  HotSpot
+//
+//  Created by aaronm17 on 7/18/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "EditInformationViewController.h"
+#import "Parse/Parse.h"
+
+@interface EditInformationViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *fullName;
+@property (weak, nonatomic) IBOutlet UITextField *username;
+@property (weak, nonatomic) IBOutlet UITextField *email;
+@property (weak, nonatomic) IBOutlet UITextField *phoneNumber;
+@property (strong, nonatomic) PFUser *currentUser;
+@property (strong, nonatomic) UIAlertController *alert;
+
+@end
+
+@implementation EditInformationViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.currentUser = [PFUser currentUser];
+    [self.currentUser saveInBackground];
+    self.fullName.text = self.currentUser[@"name"];
+    self.phoneNumber.text = self.currentUser[@"phone"];
+    self.email.text = self.currentUser[@"email"];
+    self.username.text = self.currentUser.username;
+    
+    self.alert = [UIAlertController alertControllerWithTitle:@""
```

Comment:
formatting looks off

Reply:
could you have a more descriptive name than alert? what is it for?

Reply:
could you have a more descriptive name than alert? what is it for? there's also no title.